doc/lsst.meas.extensions.shapeHSM/hsm-shape.rst

```diff
@@ -0,0 +1,42 @@
+.. py:currentmodule:: lsst.meas.extensions.shapeHSM
+
+PSF-correction algorithms
```

Comment:
I'd use some other title here: this is the text that becomes the topic link text in the main table of contents, and it didn't make any sense to me why PSF-correction was listed there. Maybe just "HSMShape Plugins" here?